### PR TITLE
docs: use `defineLazyHandler` in wasm example

### PR DESCRIPTION
**Pull request by valcosmos**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: examples/experimental/wasm/server/api/sum.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
-import { useQuery, lazyHandle } from 'h3'
+import { useQuery, defineLazyHandler } from 'h3'
 
-export default lazyHandle(async () => {
-  const { exports: { sum } } = await loadWasmInstance(
+export default defineLazyHandler(async () => {
+  const {
+    exports: { sum }
+  } = await loadWasmInstance(
```
Comment (danielroe) [inline examples/experimental/wasm/server/api/sum.ts:6]: Let's keep formatting.

```suggestion
  const { exports: { sum } } = await loadWasmInstance(
```

File: examples/experimental/wasm/server/api/sum.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import { useQuery, lazyHandle } from 'h3'
+import { useQuery, defineLazyHandler } from 'h3'
```
Comment (danielroe) [inline examples/experimental/wasm/server/api/sum.ts:1]: `useQuery` is auto-imported.

```suggestion
import { defineLazyHandler } from 'h3'
```

**Review by danielroe — COMMENTED** [review]
Thanks!

Would you also update the return so it's not using `req` but `event`?

**Comment by valcosmos** [comment]
> Thanks!
> 
> Would you also update the return so it's not using `req` but `event`?



> Thanks!
> 
> Would you also update the return so it's not using `req` but `event`?



Thank you for your reply, and I have replaced req with event.

**Comment by valcosmos** [comment]
> Thanks!
> 
> Would you also update the return so it's not using `req` but `event`?

Thank you for your reply, and I have replaced req with event.

**Comment by valcosmos** [comment]
> Thanks!
> 
> Would you also update the return so it's not using `req` but `event`?

Thanks for your reply, and I have replaced req with event.

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks!

**Pull request merged**